### PR TITLE
Add mobile menu

### PR DESCRIPTION
**Pull request by vladanvi99**
In this pull request, we added a menu for mobile.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -49,28 +57,68 @@ header nav ul {
...
-  color: #6070ff;
+header nav .mobileMenu {
+  display: flex;
+  position: fixed;
+  z-index: 99;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-image: url("../images/Overlay.png");
```
Comment (mricanho) [inline css/main.css:68]: - [ ] [OPTIONAL] The original design is a background color, not an image, and has an effect overlay, you can achieve this with `mix-blend-mode: multiply;`

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @vladanvi99 @hiromataba!, 

Good job so far! 🔥🔥 You followed most of the original design and your JS code is good and readable.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [x] Right now you only have two linters, Eslint and Stylelint, you need to add the Webhint linter on your linters.yml file, check [this link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) for instructions.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

almost there

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/main.css (hunk 1 above)
Comment (vladanvi99) [reply css/main.css:68]: image
I tried it, but it doesn't work.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**